type/__package_apt/gencode-remote

```diff
@@ -60,11 +60,21 @@ aptget="DEBIAN_FRONTEND=noninteractive apt-get --quiet --yes -o Dpkg::Options::=
 case ${state_should}
 in
     present)
-        if test -n "${version_should}" -a "${version_should}" != "${version_is}"
+        if [ -f "$__object/parameter/source" ]
+        then
+            source_path="$(cat "$__object/parameter/source")"
+            tempfile="$($__remote_exec $__target_host "mktemp -t cidst.XXXXX.${source_path##*/}")"
```

Comment:
I don't consider executing `$__remote_*` commands from a `gencode-*` script good style, as they will also be executed for dry runs.  
This section should go to a `code-local` or, even better, use `__file` if possible.
Supporting `--source -`, like `__file` would also be nice, IMO.

also: `cidst` -> `cdist`

Reply:
Well, that's an interesting and important aspect. Thanks for pointing it out.

However, I am in a bit of a pickle here:
* Putting it into `gencode-local` wont' really fly here: that will return a _filename_ (and/or create a file) that's _unique_ _locally_.
* Putting it into `__file` would then mean, we need to pass the value from the `manifest` to the `gencode-remote` script, which then means setting up temporary files and such, right?

Supporting `--sourece -` seems to be rather out of scope here: how many people will pipe a `.dpkg` as a string literal through here?

Typo is a nice catch, thanks.